Formula/pdns.rb

```diff
@@ -52,6 +54,7 @@ def install
     system "./configure", *args
 
     system "make", "install"
+    mkdir "#{var}/log/#{name}"
```

Comment:
Hardcode the `#{name}` here, it'll make it easier to follow. Also `(var/"log/pdns").mkpath` is preferable syntax, thanks!